init.sh

```diff
@@ -112,7 +119,13 @@ then
     echo "DONE"
 
     echo "===> Starting backup..."
-    exec su -pc "mydumper ${CLI_OPTIONS}" ${USER}
+    if [[ "${TARBALL^^}" != "" ]]
+    then
+        su -pc "mydumper ${CLI_OPTIONS}" ${USER}
```

Comment:
Correct me if I'm wrong, but running commands without `exec` is suboptimal, as you can't properly stop/kill the container… Could we probably chain the `tar` command via pipe or `&&`?